packages/react-components/source/react/library/alerts/Alerts.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Main content where a collection of Alert components can be passed */
+  children: PropTypes.node,
+};
+const defaultProps = {
+  children: '',
+};
+
+/** Alerts is a container for floating "growl" notifications. */
+const Alerts = ({ children }) => <div className="rc-alerts">{children}</div>;
```

Comment:
Should we possibly add role=alert and aria-live="assertive" here? - looking at what's recommended for toast/growl notifications